python-mini-projects/projects/Convert_JPEG_to_PNG/converter_GUI.py

```diff
@@ -0,0 +1,40 @@
+import tkinter as tk
+from tkinter import filedialog
+from PIL import Image
+root = tk.Tk()   # Tkinter window initialized
+root.title('Converter')     # Title of the window
+canvas1 = tk.Canvas(root, width=300, height=250, bg='orange', relief='raised')
+canvas1.pack()
+label1 = tk.Label(root, text='File Converter', bg='lightsteelblue2')   # giving a title to the screen
+label1.config(font=('helvetica', 20))
+canvas1.create_window(150, 60, window=label1)
+im1 = None  # variable to store path of image
+
+
+def getJPG():
+    '''Function to get image location and open it with pillow'''
+    global im1
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Global variables are dangerous because they can be simultaneously accessed from multiple sections of a program. This frequently results in bugs. Most bugs involving global variables arise from one function reading and acting on the value of a global variable before another function has the chance to set it to an appropriate value. We recommend using a configuration module to mutate global state.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)